lib/settings.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "ettin"
```

Comment:
Not a real objection, but it does not appear there is a current maintainer of Ettin.

Reply:
True. The canonical homes are http://github.com/mlibrary/ettin and http://github.com/mlibrary/canister and we can certainly update it as needed. @botimer will also reach out to @malakai97 and make sure that we (as MLibrary) can publish new versions on rubygems as needed.